champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/knight/sword/Ride.java

```diff
@@ -74,9 +81,9 @@ public void activate(Player player, int level) {
         horse.setStyle(Horse.Style.NONE);
         AttributeInstance horseMaxHealth = horse.getAttribute(Attribute.GENERIC_MAX_HEALTH);
         if(horseMaxHealth != null) {
-            horseMaxHealth.setBaseValue(horseHealth + ((level - 1) * 5));
+            horseMaxHealth.setBaseValue(1);
         }
-        horse.setHealth(horseHealth + ((level - 1) * 5));
+        horse.setHealth(health);
```

Comment:
Considering the horse should be despawned upon any damnage, instead of setting the horse's health to a value of 1, you should probably call `Entity#remove` on the horse in the `CustomDamageEvent` you're listening for above. Right now, when your horse gets hit, it dies and has you sitting in the air waiting for it to despawn and play the death animation.